packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -689,11 +689,23 @@ export async function getOrganization(
   if (licenseKey || process.env.LICENSE_KEY) {
     // automatically set the license data based on org license key
     license = getLicense(licenseKey || process.env.LICENSE_KEY);
-    if (!license || (license.organizationId && license.organizationId !== id)) {
+    if (
+      !license ||
+      (license.organizationId && license.organizationId !== id) ||
+      (license && !license.stripeSubscription) ||
+      (license && !license.orbSubscription)
+    ) {
       try {
         license =
-          (await licenseInit(org, getUserCodesForOrg, getLicenseMetaData)) ||
-          null;
+          (await licenseInit(
+            org,
+            getUserCodesForOrg,
+            getLicenseMetaData,
+            // if license is set, but no stripe or orb subscription, force refresh
+            !license?.stripeSubscription || !license?.orbSubscription
+              ? true
+              : false
+          )) || null;
```

Comment:
This is all getting a bit hard to follow. In `CloudProUpgradeModal` when a user successfully subscribes, this code, paired with the `refreshOrganization` call means the org's subscription status updates automatically, which is handy if they're upgrading from an upgrade CTA to get a certain feature (ex: Bandits).

Reply:
Open to suggestions on if there is a better way to handle this - maybe the `redirectWithTimeout` function?

Reply:
I believe the old stripe controllers call licenseInit with force when making the subscription.  Alternatively if we have the license returned when making a new subscription, which seems reasonable, then we can have it save directly by calling `verifyAndSetServerLicenseData(license)` from within licenseUtil.ts;

Reply:
@tzjames I was able to revert these changes now that `postNewInlineSubscriptionToLicenseServer` gets the updated license from the license-server and then calls `verifyAndSetServerLicenseData`. Thank you for that suggestion - that is much cleaner than this mess on the org controller.